tests/Aspire.Dashboard.Tests/GlobalizationTests.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace Aspire.Dashboard.Tests;
+
+public class GlobalizationTests
+{
+    [Fact]
+    public void GetSupportedCultures_IncludesPopularCultures()
+    {
+        var supportedCultures = DashboardWebApplication.GetSupportedCultures();
+
+        foreach (var localizedCulture in DashboardWebApplication.LocalizedCultures)
+        {
+            Assert.Contains(localizedCulture, supportedCultures);
+        }
+
+        Assert.Contains("en-GB", supportedCultures);
```

Comment:
Add a comment that these are a couple of examples of derived cultures we expect to be available.